### PR TITLE
small formatting changes to readme

### PR DESCRIPTION
**Pull request by nuclearkatie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,56 +2,57 @@
...
         * Call material property (T,&alpha;)
-        * Call BC/IC
-        
+        * Call BC/IC (from [#14][i14])
+
       * [Build  matrix B][i18]
-      * Call material property (T,&alpha;)
-      * Call BC/IC
-      
+      * Call material property (T,&alpha;) from (from [#21][i21])
```
Comment (gonuke) [inline README.md]: FYI - you don't need to explicitly form the link for issues if you want the link text to just be `#21`.  If will automatically convert `#21` to be a link to issue 21

Comment (nuclearkatie) [reply README.md]: We weren't sure if that feature worked for the README too, or just in comments. Thanks!

Comment (gonuke) [reply README.md]: I was completely wrong!!  What I said doesn't work... hmm...

Comment (nuclearkatie) [reply README.md]: So the way it's currently written / the link I added using the same format is fine?

**Pull request merged**